fw/rbcx-coprocessor/include/BuzzerController.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include "Bsp.hpp"
+
+void buzzerSetState(bool on) { pinWrite(buzzerPin, on); }
```

Comment:
Chtěl jsem udržet pojmenování funkcí, co jsou volané z Dispatcheru - buzzerDispatch - co myslíš?

Reply:
Původně jsem to tak měl, ale pak mi přišlo pitomé mít druhou funkci, ve které jediný rozdíl je, že vybere jeden book z té PB struktury. Toto bylo koncipováno jakože to může být voláno jak z dispatcheru, tak z nějakého kódu v STM (signalizace vybití baterek?)